admiral/pkg/controller/admiral/service.go

```diff
@@ -41,6 +42,11 @@ type serviceCache struct {
 }
 
 func (s *serviceCache) Put(service *k8sV1.Service) {
+	if service.Labels[common.GetLabelSet().AdmiralIgnoreLabel] == "true" {
```

Comment:
Can reuse the approach on deployments, https://github.com/istio-ecosystem/admiral/blob/70af3fb2459102bc31e47ecb1307b2e342435dab/admiral/pkg/controller/admiral/deployment.go#L185

Given we are not using labels for client side filters, it would be best to keep stuff in annotations and consistent